src/variant.ts

```diff
@@ -46,6 +52,16 @@ function getSeed(context: Context): string {
     return result || String(Math.round(Math.random() * 100000));
 }
 
+function overrideMatchesContext(context: Context): any {
```

Comment:
@sveisvei Do you know of to define the return type here: "a Function which returns a boolean"

Reply:
```
function overrideMatchesContext(context: Context): (a: string) => boolean {
}
```